flink-end-to-end-tests/pom.xml

```diff
@@ -153,6 +170,32 @@ under the License.
 
 	<build>
 		<plugins>
+			<plugin>
+				<artifactId>maven-resources-plugin</artifactId>
+				<!-- <version>3.1.0</version> -->
```

Comment:
?

Reply:
I think I copy pasted the plugin definition from somewhere and assumed that the plugin version is centrally managed.

The other uses of the plugin are also defined without a version
```
--
flink-quickstart/pom.xml-			<plugin>
flink-quickstart/pom.xml-				<groupId>org.apache.maven.plugins</groupId>
flink-quickstart/pom.xml:				<artifactId>maven-resources-plugin</artifactId>
flink-quickstart/pom.xml-				<configuration>
flink-quickstart/pom.xml-					<useDefaultDelimiters>false</useDefaultDelimiters>
--
flink-table/flink-sql-parser-hive/pom.xml-			<!-- adding fmpp code gen -->
flink-table/flink-sql-parser-hive/pom.xml-			<plugin>
flink-table/flink-sql-parser-hive/pom.xml:				<artifactId>maven-resources-plugin</artifactId>
flink-table/flink-sql-parser-hive/pom.xml-				<executions>
flink-table/flink-sql-parser-hive/pom.xml-					<execution>
--
flink-table/flink-sql-parser/pom.xml-			<!-- adding fmpp code gen -->
flink-table/flink-sql-parser/pom.xml-			<plugin>
flink-table/flink-sql-parser/pom.xml:				<artifactId>maven-resources-plugin</artifactId>
flink-table/flink-sql-parser/pom.xml-				<executions>
flink-table/flink-sql-parser/pom.xml-					<execution>
--
flink-walkthroughs/pom.xml-			<plugin>
flink-walkthroughs/pom.xml-				<groupId>org.apache.maven.plugins</groupId>
flink-walkthroughs/pom.xml:				<artifactId>maven-resources-plugin</artifactId>
flink-walkthroughs/pom.xml-				<configuration>
flink-walkthroughs/pom.xml-					<useDefaultDelimiters>false</useDefaultDelimiters>
```

It doesn't seem that we are centrally managing the plugin versions. I would propose to just remove the `<!-- <version>3.1.0</version> -->` line.

Reply:
yes we should remove it. It's a bit odd that we don't have a managed version, but that's a separate issue.